lib/package.rb

```diff
@@ -36,7 +36,10 @@ def self.load_package(pkg_file)
     pkg_name = File.basename(pkg_file, '.rb')
     class_name = pkg_name.capitalize
 
-    # read and eval package script under 'Package' class
+    # Read and eval package script under 'Package' class, using the
+    # newest file available.
+    pkg_file = File.file?("#{CREW_LOCAL_REPO_ROOT}/packages/#{pkg_name}.rb") ? ["#{CREW_LOCAL_REPO_ROOT}/packages/#{pkg_name}.rb", "#{CREW_PACKAGES_PATH}/#{pkg_name}.rb"].max { |a, b| File.mtime(a) <=> File.mtime(b) } : "#{CREW_PACKAGES_PATH}/#{pkg_name}.rb"
```

Comment:
Simplify with `Dir.glob`?
```ruby
# Dir[] will also check the existence of file
pkg_file = Dir["/{#{CREW_LOCAL_REPO_ROOT}/packages,#{CREW_PACKAGES_PATH}}/#{pkg_name}.rb"].max { |a, b| File.mtime(a) <=> File.mtime(b) }
```

Reply:
That doesn't work when `CREW_LOCAL_REPO_ROOT` is empty.

```ruby
pkg_name = 'ninja'
CREW_LOCAL_REPO_ROOT = '/output/git/satmandu'
CREW_PACKAGES_PATH = '/usr/local/lib/crew/packages'
Dir["#{CREW_LOCAL_REPO_ROOT}/packages/#{pkg_name}.rb","#{CREW_PACKAGES_PATH}}/#{pkg_name}.rb"].max { |a, b| File.mtime(a) <=> File.mtime(b) }
```
gives:
```ruby
=> "/output/git/satmandu/packages/ninja.rb"
```
But
```ruby
pkg_name = 'ninja'
CREW_LOCAL_REPO_ROOT = ''
CREW_PACKAGES_PATH = '/usr/local/lib/crew/packages'
Dir["#{CREW_LOCAL_REPO_ROOT}/packages/#{pkg_name}.rb","#{CREW_PACKAGES_PATH}}/#{pkg_name}.rb"].max { |a, b| File.mtime(a) <=> File.mtime(b) }
```
gives:
```ruby
=> nil
```

Reply:
You have an excess bracket here...
```ruby
"#{CREW_PACKAGES_PATH}>>>}<<</#{pkg_name}.rb"
```